From 534b143709d6fd89f36710fc880a350acc6d5829 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@foss.st.com>
Date: Fri, 4 Oct 2024 17:47:26 +0200
Subject: [PATCH 1261/1695] drivers: clk: change tree trace level

Change clock print tree trace level so that
it can bee seen when it is requested by xtest.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git e4910c3541ba98b06b276a7f0b79ec13183c3438 drivers: clk: change tree trace level]
Change-Id: I3f6c9165bde2c8b25200343f089f0d00688f0949
Signed-off-by: Pascal Paillet <p.paillet@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/409232
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
Tested-by: Pascal PAILLET <p.paillet@st.com>
Reviewed-by: Pascal PAILLET <p.paillet@st.com>
---
 core/drivers/clk/clk.c | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/core/drivers/clk/clk.c b/core/drivers/clk/clk.c
index 62353adba..718b2f636 100644
--- a/core/drivers/clk/clk.c
+++ b/core/drivers/clk/clk.c
@@ -612,7 +612,7 @@ out:
 	if (!msg)
 		snprintf(msg_end - 4, 4, "...");
 
-	DMSG("%s", msg_buf);
+	IMSG("%s", msg_buf);
 }
 
 static void print_tree(void)
@@ -624,7 +624,7 @@ static void print_tree(void)
 
 #ifdef CFG_DRIVERS_CLK_PRINT_TREE
 	if (SLIST_EMPTY(&clock_list)) {
-		DMSG("-- No registered clock");
+		IMSG("-- No registered clock");
 		return;
 	}
 #endif
@@ -657,9 +657,8 @@ static void print_tree(void)
 
 void clk_print_tree(void)
 {
-	if (IS_ENABLED(CFG_DRIVERS_CLK_PRINT_TREE) &&
-	    TRACE_LEVEL >= TRACE_DEBUG) {
-		DMSG("Clock tree summary (informative):");
+	if (IS_ENABLED(CFG_DRIVERS_CLK_PRINT_TREE)) {
+		IMSG("Clock tree summary (informative):");
 		print_tree();
 	}
 }
-- 
2.47.2

